### ISSUE TITLE
handle memory items better

### ISSUE THREAD

**Issue by Icehunter** [Icehunter/sharlayan#8]
@cybaersam Adding you here because you were working on a fix in the radar.

https://github.com/Icehunter/ffxivapp-memory/blob/master/FFXIVAPP.Memory/Reader.Actor.cs#L81

That is where I start looking at whether something was in the list previously or not; and determine which ones are new.

The main application fires the OnNewEntities events only 1 time; and sends a concurrentDictionary by ref. I think if radar was to listen to removed/added events it could handle that?

**Comment by Icehunter** [comment]
Going to test this out now:

https://github.com/Icehunter/ffxivapp-memory/commit/80d73cdc3b9e4ef3dff4780539ff958e575b1ea2

**Comment by cybaersam** [comment]
well if this works then  we can remove the code from the radar ^_^

**Comment by vekien** [comment]
Not really, the Radar would still see dead enemies if they load in memory, which rank ones do as far as I'm aware?

**Comment by cybaersam** [comment]
true.... though... if i remember correctly before heavensward they disappeared after a few seconds
haven't been very active with the app for a while when heavensward was released till now...

so if Icehunter's fix here  gets that functionality back it should render my temp fix unnecessary in the radar...
well we will have to test it and see how it affects  the radar and the info in the dictionaries

my goal is after this will be for the palace of the dead to better find hidden treasures (if that is even possible... i'm not sure if this information is in ram atleast i know its not in the dictionaries) and traps...
though for this i will need to be able to zoom in the radar... will try to get that after this is fixed ^_^
